File: Chapter9/Java/Using Java's Built-in Methods in Java/ChangeCase.java

// ChangeCase.java - This program converts a string to lowercase and uppercase.
// Input:  Interactive
// Output:  Uppercase and lowercase versions of the user-entered string.
import java.util.Scanner;

public class ChangeCase {
    public static void main(String args[]) {
        Scanner s = new Scanner(System.in);
        String sample;
        String result;

        System.out.println("Enter a string or done when you want to quit.");

        sample = s.nextLine();

        while (sample.compareTo("done") != 0) {
            // Call () method here and print the result.
            result = sample.toLowerCase();

            System.out.println("Lowercase: " + result);
            // Call () method here and print the result.
            result = sample.toUpperCase();
            System.out.println("Uppercase: " + result);
            System.out.println("Enter a string or done when you want to quit.");
            sample = s.nextLine();
        }

        System.exit(0);

    } // End of main() method.

} // End of ChangeCase class.

File: Chapter9/Python/Using Python's Built-in Methods in Python/ChangeCase.py
"""
ChangeCase.py - This program converts a string to lowercase and uppercase.
Input:  Interactive
Output:  Uppercase and lowercase versions of the user-entered string.
"""
def lower(sample):
    #Change lowercase
    return sample.lower()

def upper(sample):
    #Change uppercase
    return sample.upper()

sample = input("Enter a string or done when you want to quit: ")

while sample != "done":
    # Call the lower method here and print the result.

    result = lower(sample)

    print("Lowercase: " + result)
    # Call upper method here and print the result.

    result = upper(sample)

    print("Uppercase: " + result)
    sample = input("Enter a string or done when you want to quit: ")




